docs/csharp/language-reference/tokens/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "C# Tokens (C# Reference) | Microsoft Docs"
+ms.date: "2017-02-14"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+f1_keywords: 
+  - "cs.tokens"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "tokens [C#]"
+  - "C# language, tokens"
+  - "Visual C#, tokens"
+  - "@ token (C#)"
+  - "@ token (C#)"
+ms.assetid: 4c5c0539-2e37-40b7-91ce-75af5aabd3f9
+caps.latest.revision: 22
+author: "rpetrusha"
+ms.author: "ronpet"
+translation.priority.ht: 
+  - "cs-cz"
+  - "de-de"
+  - "es-es"
+  - "fr-fr"
+  - "it-it"
+  - "ja-jp"
+  - "ko-kr"
+  - "pl-pl"
+  - "pt-br"
+  - "ru-ru"
+  - "tr-tr"
+  - "zh-cn"
+  - "zh-tw"
+---
+
+# C# Tokens
+
+Tokens are predefined, contextual, single-character symbols, such as `@` and `$`, that have a special meaning to the C# compiler. C# supports the following tokens: 
```

Comment:
This is really misleading.  A 'token' in the lexical analysis could be any of:
an identifier
a keyword
literals
operators
punctuation

These two, the `@` an the `$` are kind of special. The spec calls them out as special cases of literals that start with the right character: https://github.com/dotnet/csharplang/blob/master/spec/lexical-structure.md#string-literals

So, I would refer to these characters as "tokens", but I wouldn't add a page for 'tokens' that includes only these characters.  I'm sorry if I misled you earlier in our meeting.